contracts/truefi2/LoanToken2.sol

```diff
@@ -150,7 +150,7 @@ contract LoanToken2 is ILoanToken2, ERC20 {
         uint256 _amount,
         uint256 _term,
         uint256 _apy
-    ) public {
+    ) external initializer {
```

Comment:
Shouldn't the contract be `Initializable` in order for that to work?

Reply:
Ok, my bad, all good 😅

Reply:
ERC20 is Initializable (it wouldn't even compile if it wasn't)